bot/db_lotto_bot.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE lotto_bot_control (
+    lotto_bot_control_id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    status VARCHAR(10) NOT NULL, 
+    device_address CHAR(33) NOT NULL, 
+    last_update TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+INSERT INTO lotto_bot_control (status, device_address) VALUES ('off', 'init');
+
+CREATE TABLE lotto_bot_player (
+    lotto_bot_player_id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    nick_name VARCHAR(50) NOT NULL, 
```

Comment:
not sure if we need player info, everything could be based on device id. Simple and anonymous just like real lottery. We can leave it for now though.